test/dlc_tests/ec2/tensorflow/training/test_tensorflow_training.py

```diff
@@ -201,7 +201,7 @@ def test_tensorflow_addons_cpu(tensorflow_training, ec2_connection, tf2_only, cp
 # Helper function to test data service
 def run_data_service_test(ec2_connection, tensorflow_training, cmd):
     ec2_connection.run('python3 -m pip install --upgrade pip')
-    ec2_connection.run('pip3 install tensorflow==2.4')
+    ec2_connection.run('pip3 install tensorflow==2.5.0-rc3')
```

Comment:
this needs to be changed to 2.5.0 since we are not on rcs anymore

Reply:
Done